src/components/AssetInput/AssetThumbnail.tsx

```diff
@@ -0,0 +1,35 @@
+import IconAudio from "@foundation/Icon/Generated/IconAudio";
+import { IconSize } from "@foundation/Icon/IconSize";
+import { merge } from "@utilities/merge";
+import React, { cloneElement, FC } from "react";
+import { AssetInputProps, AssetInputSize } from "./AssetInput";
+import { SelectedAssetProps } from "./SingleAsset/SelectedAsset";
+
+export type AssetThumbnailProps = {
+    asset: SelectedAssetProps["asset"];
+    size: AssetInputProps["size"];
+    isActive?: boolean;
+};
+export const AssetThumbnail: FC<Required<AssetThumbnailProps>> = ({ asset, size, isActive }) => (
+    <div
+        className={merge([
+            "tw-flex tw-flex-none tw-items-center tw-justify-center tw-bg-black-5 dark:tw-bg-black-95",
+            size === AssetInputSize.Large ? "tw-w-full tw-h-32" : "tw-w-14 tw-h-full",
```

Comment:
The thumbnail in the multi asset input is smaller according to the design (45px). I suggest you use `tw-w-11` (44px) instead of `tw-w-14` if it's a multi asset input

Reply:
done